cfe_ts_crc.c

```diff
@@ -102,20 +103,21 @@ uint32 CalculateCRC(void *DataPtr, uint32 DataLength, uint32 InputCRC)
 
 int main(int argc, char **argv)
 {
-    int    readSize;
-    int    skipSize = 0;
-    int    fileSize = 0;
-    uint32 fileCRC  = 0;
-    int    fd;
-    int    done = 0;
-    char   buffer[100];
+    ssize_t readSize;
+    off_t   skipSize = 0;
+    ssize_t fileSize = 0;
+    uint32  fileCRC  = 0;
+    int     fd;
+    char    buffer[100];
+    off_t   offsetReturn = 0;
 
     /* check for valid input */
     if ((argc != 2) || (strncmp(argv[1], "--help", 100) == 0))
     {
         printf("%s\n", CFE_TS_CRC_VERSION_STRING);
         printf("\nUsage: cfe_ts_crc [filename]\n");
-        exit(0);
+        printf("%s\n", strerror(errno));
+        exit(1);
```

Comment:
Sorry, when I said to print the error whenever doing `exit(1)` I meant when it is due to an actual error.  This case it is a user error, not an errno from the C library.  Don't print the errno here, it's not relevant.

Reply:
fixed